### ISSUE TITLE
forge inspect <sc> abi --pretty raises error

### ISSUE THREAD

**Issue by radeksvarz** [foundry-rs/foundry#6816]
### Component

Forge

### Have you ensured that all of these are up to date?

- [X] Foundry
- [ ] Foundryup

### What version of Foundry are you on?

forge 0.2.0 (c5fd67b 2024-01-15T00:23:11.184513736Z)

### What command(s) is the bug in?

forge inspect

### Operating System

Linux

### Describe the bug

```
💤 forge inspect src/Counter.sol:Counter abi --pretty
Error: 
an error occurred when formatting an argument
```

Raises also on the very basic contract after forge init.

**Comment by mattsse** [comment]
thanks for flagging,
smol bug, fixed in #6817

**Comment by radeksvarz** [comment]
ok, as of `forge 0.2.0 (03f5a95 2024-01-17T16:46:46.891738603Z)`

it works now:
```

💤 forge inspect src/Counter.sol:Counter abi --pretty
interface Counter {
    function increment() external;
    function number() external view returns (uint256);
    function setNumber(uint256 newNumber) external;
}


```
